### PR TITLE
Update registry URLs to match IBS releases at suse.com

### LINKED ISSUE #178: Update our docs to comsume all artifacts from IBS

### PR DESCRIPTION
**Pull request by davidcassany**
Fixes #178

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ SLE Micro for Rancher is a containerized and "stripped to the bones" OS layer. A
 It's sole purpose is to run Kubernetes (k3s or RKE2), with everything controlled through Rancher Manager.
 
 Elemental Teal is built in the [openSUSE Build Service](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Stable:Teal53/node-image)
-and available through the [openSUSE Registry](https://registry.opensuse.org/cgi-bin/cooverview?srch_term=project%3D%5Eisv%3ARancher%3AElemental%3AStable+container%3D.*).
+and available through the [SUSE Registry](https://registry.suse.com).
```
Comment (davidcassany) [inline docs/index.md:32]: I am doubtful about this one. If you navigate to `registry.suse.com` from your browser it is not possible to search and find Elemental in there in any way (at least I couldn't). So I am not we should add this link here, probably we could add something like:

```
... and available through the SUSE registry ('registry.suse.com').
```
without an http url.

File: versioned_docs/version-1.2/index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ SLE Micro for Rancher is a containerized and "stripped to the bones" OS layer. A
 It's sole purpose is to run Kubernetes (k3s or RKE2), with everything controlled through Rancher Manager.
 
 Elemental Teal is built in the [openSUSE Build Service](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Stable:Teal53/node-image)
-and available through the [openSUSE Registry](https://registry.opensuse.org/cgi-bin/cooverview?srch_term=project%3D%5Eisv%3ARancher%3AElemental%3AStable+container%3D.*).
+and available through the [SUSE Registry](https://registry.suse.com).
```
Comment (davidcassany) [inline versioned_docs/version-1.2/index.md:32]: Same as above, I doubt this is a good way to link it.

**Comment by davidcassany** [comment]
I also noted we are not providing the helm commands to install the operator for the stable releases in our quickstart guide, we just provide it for the staging (RPi) and dev environments. I think we should also provide the commands there too.

File: docs/index.md (hunk 1 above)
Comment (kkaempf) [reply docs/index.md:32]: Aren't there CLI tools available to browse a registry ? These should work with registry.suse.com 🤔

**Review by kkaempf — APPROVED** [review]
Seems ok (haven't validated all the URLs, though 😉 )

File: docs/elementaloperatorchart-reference.md [hunk 3]
```diff
@@ -17,15 +17,15 @@ The associated chart bootstraps an elemental-operator deployment on the [Rancher
...
-helm pull oci://registry.opensuse.org/isv/rancher/elemental/stable/charts/rancher/elemental-operator-chart
-helm show all oci://registry.opensuse.org/isv/rancher/elemental/stable/charts/rancher/elemental-operator-chart
+helm pull oci://registry.suse.com/rancher/elemental-operator-chart
+helm show all oci://registry.suse.com/rancher/elemental-operator-chart
 ```
 
 ## Install Chart
 
 ```console showLineNumbers
 helm install --create-namespace -n cattle-elemental-system elemental-operator \
```
Comment (fgiudici) [inline docs/elementaloperatorchart-reference.md:27]: Here (and few lines above) we have not added the crds chart... probably we have to in another issue/PR as this whole docs section should be reviewed

File: docs/index.md (hunk 1 above)
Comment (fgiudici) [reply docs/index.md:32]: Why anyway drop the openSUSE registry link? I would add the suse.com but leave the opensuse one too

File: versioned_docs/version-1.2/partials/_elemental-operator-install.md [hunk 4]
```diff
@@ -5,6 +5,11 @@ cluster and taking care of creating inventories, registrations for machines and
 
 We will use the Helm package manager to install the elemental-operator chart into our cluster.
 
+```shell showLineNumbers
+helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator-crds oci://registry.suse.com/rancher/elemental-operator-crds-chart
+helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator oci://registry.suse.com/rancher/elemental-operator-chart
+```
```
Comment (fgiudici) [inline versioned_docs/version-1.2/partials/_elemental-operator-install.md:11]: Shouldn't we target a specific chart version in the versioned docs (i.e., --version 1.2.5 parameter)? At least in the installation instruction, for the reference and troubleshooting we may leave the *unversioned* ones.

File: versioned_docs/version-1.2/index.md (hunk 2 above)
Comment (fgiudici) [reply versioned_docs/version-1.2/index.md:32]: I would leave also the OBS reference as stated above...
apart from this, what about giving the link to the teal image repo in the registry?
oci://registry.suse.com/rancher/elemental-teal/5.3

One could inspect / list the tags at least, e.g., with skopeo:
`skopeo list-tags docker://registry.suse.com/rancher/elemental-teal/5.3`
the oci link may prevent misunderstandings.

File: versioned_docs/version-1.2/partials/_elemental-operator-install.md [hunk 5]
```diff
@@ -5,6 +5,11 @@ cluster and taking care of creating inventories, registrations for machines and
 
 We will use the Helm package manager to install the elemental-operator chart into our cluster.
 
+```shell showLineNumbers
+helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator-crds oci://registry.suse.com/rancher/elemental-operator-crds-chart
+helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator oci://registry.suse.com/rancher/elemental-operator-chart
```
Comment (fgiudici) [inline versioned_docs/version-1.2/partials/_elemental-operator-install.md:10]:
```suggestion
helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator --version 1.2.5 oci://registry.suse.com/rancher/elemental-operator-chart
```

File: versioned_docs/version-1.2/partials/_elemental-operator-install.md [hunk 6]
```diff
@@ -5,6 +5,11 @@ cluster and taking care of creating inventories, registrations for machines and
 
 We will use the Helm package manager to install the elemental-operator chart into our cluster.
 
+```shell showLineNumbers
+helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator-crds oci://registry.suse.com/rancher/elemental-operator-crds-chart
```
Comment (fgiudici) [inline versioned_docs/version-1.2/partials/_elemental-operator-install.md:9]:
```suggestion
helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator-crds --version 1.2.5 oci://registry.suse.com/rancher/elemental-operator-crds-chart
```

**Review by fgiudici — APPROVED** [review]
LGTM!

File: versioned_docs/version-1.2/index.md (hunk 2 above)
Comment (davidcassany) [reply versioned_docs/version-1.2/index.md:32]: OBS reference is good, as it points to the source code. Listing available tags from an skopeo/reg call sounds good to me too. Probably `reg` as it is also available for Mac, Windows and Linux.

**Review by nunix — APPROVED** [review]

**Pull request merged**